lib/githubchart/svg.rb

```diff
@@ -21,6 +22,14 @@ def svg
       chart.to_s
     end
 
+    def svg_square
+      grid = matrix.minor(0..(matrix.row_size-1), (matrix.column_size - 7)..(matrix.column_size - 1))
```

Comment:
Admit I don't totally understand the Ruby magic that goes into building this matrix. Seems to me this method is only called once so performance should not be an issue. But maybe there's something I'm missing.